schemas/openshift/terraform-resource-1.yml

```diff
@@ -131,6 +131,15 @@ properties:
     type: array
     items:
       type: object
+  publish_log_types:
+    type: array
+    items:
+      type: string
+      enum:
```

Comment:
you can probably leave out the enum in this section to avoid duplication?

Reply:
I thought so too, but sadly it doesn't work if I leave this out:

```
"reason": "VALIDATION_ERROR",
"error": "Additional properties are not allowed ('publish_log_types' was unexpected)
```

Reply:
I meant only the enum, not the entire thing

Reply:
Good idea!
Fixed